x-pack/plugins/security/public/components/form_flyout.tsx

```diff
@@ -59,46 +58,44 @@ export const FormFlyout: FunctionComponent<FormFlyoutProps> = ({
   const titleId = useHtmlId('formFlyout', 'title');
 
   return (
-    <EuiPortal>
-      <EuiFlyout onClose={onCancel} aria-labelledby={titleId} {...rest}>
-        <EuiFlyoutHeader hasBorder>
-          <EuiTitle size="m">
-            <h2 id={titleId}>{title}</h2>
-          </EuiTitle>
-        </EuiFlyoutHeader>
-        <EuiFlyoutBody>{children}</EuiFlyoutBody>
-        <EuiFlyoutFooter>
-          <EuiFlexGroup justifyContent="spaceBetween">
+    <EuiFlyout onClose={onCancel} aria-labelledby={titleId} {...rest}>
```

Comment:
Are these changes necessary? Does `api_key_flyout` use `FormFlyout`? Or is this to just keep things consistent?
Note: I couldn't find any uses of `FormFlyout`.